ql/termstructures/yield/ultimateforwardtermstructure.hpp

```diff
@@ -149,6 +169,25 @@ namespace QuantLib {
         }
     }
 
+    inline Rate UltimateForwardTermStructure::applyRounding(Rate r, Time t) const {
+        if (!rounding_.has_value()) {
+            return r;
+        }
+        // Input rate is continuously compounded by definition.
+        // Hence, in case this is also the selected compounding method for rounding,
+        // it is not required to calculate equivalent rates, and rounding
+        // may be applied directly.
+        Rate equivalentRate = isCompoundingContinuous_ ?
```

Comment:
We can probably write `compounding == Continuous` without having to declare and initialize a data member.